README.md

```diff
@@ -1,75 +1,93 @@
-Rundeck
-========
+<p align="center">
+<a href="https://www.rundeck.com#gh-light-mode-only">
+  <img src="https://www.rundeck.com/hubfs/Pager%20Duty%20Branding/RundeckbyPagerDuty.svg" alt="Rundeck" width="300"/>
+</a>
+<a href="https://www.rundeck.com#gh-dark-mode-only">
+  <img src="https://www.rundeck.com/hubfs/Pager%20Duty%20Branding/RundeckbyPagerDutyDM.svg" alt="Rundeck" width="300"/>
+</a>
```

Comment:
it seems this is not the preferred way to have dark-mode/light-mode images: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to